synthtool/gcp/templates/node_library/.github/auto-approve.yml

```diff
@@ -0,0 +1,12 @@
+rules:
+- author: "release-please[bot]"
+  title: "^chore: release"
+  changedFiles:
+  - "package\\.json$"
+  - "CHANGELOG\\.md$"
+  maxFiles: 3
+- author: "gcf-owl-bot[bot]"
```

Comment:
I think we should do the `.OwlBot.lock` config in a separate PR. I'm personally comfortable with automating merging these files. But @chingor13 had some concerns we should talk through as a group.